lighthouse-core/gather/gatherers/styles.js

```diff
@@ -98,8 +99,15 @@ class Styles extends Gatherer {
         }).then(content => {
           const styleHeader = this._activeStyleHeaders[sheetId];
           styleHeader.content = content.text;
-          styleHeader.parsedContent = getCSSPropsInStyleSheet(
-              parser.parse(styleHeader.content));
+
+          const parsedContent = parser.parse(styleHeader.content);
+          if (parsedContent.error) {
+            log.warn('Styles Gatherer', '....${parsedContent.error}');
```

Comment:
> `'....${parsedContent.error}'`

no idea why I had that `....` there :) Maybe I meant `` `something something: ${parsedContent.error}` ``? As this is, it won't actually print the error